assignments/models.py

```diff
@@ -0,0 +1,149 @@
+from django.db import models
+from django.utils.text import slugify
+from django.utils.translation import ugettext as _
+
+
+class Assignment(models.Model):
+    """
+    Assignment is a top level concept of the application.
+    Every assignment consists of several tasks.
+    It is uniquely identified by specified name and area.
+    """
+
+    STATUS_CHOICES = (
+        ('open', _('Open')),
+        ('closed', _('Closed'))
+    )
+
+    name = models.CharField(_('name'), max_length=128)
+    area = models.CharField(_('area'), max_length=256)
+    status = models.CharField(_('status'), max_length=128, choices=STATUS_CHOICES)
+    budget = models.IntegerField(_('budget'), default=0)
+    slug = models.CharField(max_length=128, unique=True)
+
+    class Meta:
+        verbose_name = _('assignment')
+        verbose_name_plural = _('assignments')
+        unique_together = ('name', 'area')
+
+    def save(self, *args, **kwargs):
+        if self.slug is None:
+            self.slug = slugify('{}{}'.format(self.name, self.area)[:100])
+        return super(Assignment, self).save(*args, **kwargs)
+
+    def __str__(self):
+        return self.name
+
+
+class Student(models.Model):
+    """
+    Learning environment user
+    """
+    SEX_CHOICES = (
+        ('male', _('Male')),
+        ('female', _('Female'))
+    )
+    school = models.CharField(_('school name'), max_length=256)
```

Comment:
School and school's classes should define in own models. Those are good to define only once.